src/make_app.ts

```diff
@@ -49,7 +49,14 @@ export function makeApp(options: AppOptions): PolyserveApplication {
   app.get('*', function (req, res) {
     // Serve local files from . and other components from bower_components
     let url = parseUrl(req.url, true);
-    let splitPath = url.pathname.split('/').slice(1);
+    let pathPieces = url.pathname.split('/');
```

Comment:
btw, this isn't necessary, `send()` has an option to deny dotfiles: https://www.npmjs.com/package/send#dotfiles